### PR TITLE
update links

### PR DESCRIPTION
**Pull request by shamsartem**

### CODE REVIEW

File: aqua-examples/decentralized-blockchain-gateway/README.md [hunk 1]
```diff
@@ -920,7 +920,7 @@ ipfs --api "/ip4/161.35.222.178/tcp/5001/p2p/12D3KooWApmdAtFJaeybnXtf1mBz1Tukxyr
...
+Woo hoo!! All dressed up and nowhere to go ... just yet. Let's change that. In order to be able to deal with IPFS documents, we need to be able to interact with IPFS from Aqua. We could write a [custom IPFS adapter](./wasm-module/ipfs-adapter) and [custom IPFS processing module](./wasm-module/ipfs-cli) or use Fluence's builtin [IPFS integration with Aqua](https://fluence.dev/docs/aqua-book/libraries/aqua-ipfs) library. If we use the latter, we don't have to implement and deploy our own ipfs adapter service(s) and don't have to manage the associated function addresses. However, the Fluence IPFS library is geared toward file management, which means that we need to write and maintain a file processing service, which still leaves us with service management chores. So let's write and deploy a super-light ipfs adapter based on the ipfs cli[cat](https://docs.ipfs.io/reference/cli/#ipfs-cat) command.
```
Comment (boneyard93501) [inline aqua-examples/decentralized-blockchain-gateway/README.md:923]: might be better to open https://docs.ipfs.io/reference/cli/#ipfs-cat as a new tab?

File: aqua-examples/near-integration/README.md [hunk 2]
```diff
@@ -270,7 +270,7 @@ In order to create a NEAR Wasm adapter, we wrap whatever functionality we need f
 
 ### Creating And Deploying NEAR Wasm Services
 
-In the `services` directory, you find a minimal Wasm adapter for [NEAR RPC API](https://docs.near.org/docs/api/rpc) to get you started. Since we are connecting to on-chain resources via JSON-RPC, we need our service module to have access to [cUrl](https://doc.fluence.dev/docs/tutorials_tutorials/curl-as-a-service), which we provide with the [cUrl adapter](../near-integration/services/curl-adapter/):
+In the `services` directory, you find a minimal Wasm adapter for [NEAR RPC API](https://docs.near.org/docs/api/rpc) to get you started. Since we are connecting to on-chain resources via JSON-RPC, we need our service module to have access to [cUrl](https://fluence.dev/docs/build/tutorials/curl-as-a-service), which we provide with the [cUrl adapter](../near-integration/services/curl-adapter/):
```
Comment (boneyard93501) [inline aqua-examples/near-integration/README.md:273]: looks like `https://docs.near.org/docs/api/rpc` now redirects to `https://docs.near.org/api/rpc/introduction`.

File: marine-examples/README.md [hunk 3]
```diff
@@ -806,8 +806,8 @@ Your peerId: 12D3KooWBVzSqoQqFycENVhw7W5RY1UHwPCn6U9iHG2mbwuCuLq3
 "ec71a1fc-66d7-41f4-bff1-f9c07d361bd4"
 ```
 
-The (peer id, service id) tuple is going to be useful once you start putting the service to work with, say, [Aqua](https://doc.fluence.dev/aqua-book/) and you should hold on the data for future use. Also, you can look up your service on the [Fluence Dashboard](https://dash.fluence.dev/blueprint/8d210ec2b83e4c661c71820b79f02d99908794e8af8034b465762f61682bc43b).
+The (peer id, service id) tuple is going to be useful once you start putting the service to work with, say, [Aqua](https://fluence.dev/docs/aqua-book/introduction) and you should hold on the data for future use. Also, you can look up your service on the [Fluence Dashboard](https://dash.fluence.dev/blueprint/8d210ec2b83e4c661c71820b79f02d99908794e8af8034b465762f61682bc43b).
```
Comment (boneyard93501) [inline marine-examples/README.md:809]: `https://dash.fluence.dev/blueprint/8d210ec2b83e4c661c71820b79f02d99908794e8af8034b465762f61682bc43b` seems to go to nowhere. but that might just be a dashboard issue.

File: quickstart/Readme.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
 # Quick Start
 
-For the documentation of this tutorial, please see the Fluence [developer documentation](https://doc.fluence.dev/docs/quick-start).
+For the documentation of this tutorial, please see the Fluence [developer documentation](https://fluence.dev/docs/build/quick-start/).
 If you have any questions, please contact us in our[discord]( https://fluence.chat) channel, raise an issue or open a PR.
```
Comment (boneyard93501) [inline quickstart/Readme.md:4]: not a functional issue but  that extra space  in ( https://fluence.chat) probably should go.

File: aqua-examples/decentralized-blockchain-gateway/README.md (hunk 1 above)
Comment (shamsartem) [reply aqua-examples/decentralized-blockchain-gateway/README.md]: We can do that, but I think it's weird to do that for this exact link. I think all of the links in md on GitHub by default open in the same tab so it's default and expected behavior

File: marine-examples/README.md (hunk 3 above)
Comment (shamsartem) [reply marine-examples/README.md:809]: Yeah it's a dashboard issue

File: quickstart/Readme.md (hunk 4 above)
Comment (shamsartem) [reply quickstart/Readme.md]: fixed

File: aqua-examples/near-integration/README.md (hunk 2 above)
Comment (shamsartem) [reply aqua-examples/near-integration/README.md]: fixed

**Review by boneyard93501 — APPROVED** [review]

**Pull request merged**